src/Android/Xamarin.Android/Samples/GeometryEngine/CreateGeometries/README.md

```diff
@@ -0,0 +1,7 @@
+# Create geometries
```

Comment:
We should talk about the process for the readmes. Ideally we'd be using automation to create these (to save time and ensure things are consistent).

No action needed, just thinking out loud.